### ISSUE TITLE
Crash on iOS 12

### ISSUE THREAD

**Issue by anderscheow** [fluttercommunity/flutter_webview_plugin#230]
```
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000109a381bb __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x0000000108fd6735 objc_exception_throw + 48
	2   CoreFoundation                      0x0000000109a56f44 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x0000000109a3ced6 ___forwarding___ + 1446
	4   CoreFoundation                      0x0000000109a3eda8 _CF_forwarding_prep_0 + 120
	5   Runner                              0x0000000104a52930 -[FlutterWebviewPlugin initWebview:] + 2832
	6   Runner                              0x0000000104a517fe -[FlutterWebviewPlugin handleMethodCall:result:] + 254
	7   Flutter                             0x000000010574d75b __45-[FlutterMethodChannel setMethodCallHandler:]_block_invoke + 115
	8   Flutter  <…>
```

This is the code:
```
class WebViewPage extends StatelessWidget {
  final String url;
  final String title;

  WebViewPage({this.url, this.title});

  @override
  Widget build(BuildContext context) {
    return new WebviewScaffold(
      url: url,
      appBar: AppBar(
        title: Text(title),
      ),
    );
  }
}
```

**Comment by dinhha** [comment]
I also face this crash issue when upgrade to Flutter 0.10.2 beta.
It works well on Flutter 0.10.0

**Comment by klavergne** [comment]
Same on Flutter 0.10.3.

**Comment by klavergne** [comment]
@anderscheow That works, but I wonder when it will be supported on the later versions of Flutter.

**Comment by dinhha** [comment]
You can try on my fork. I fixed it
https://github.com/dinhha/flutter_webview_plugin/commit/aca88cb779d3f06988dcd4f8da7e6bc12456d17b

**Comment by slightfoot** [comment]
Fixed as of `v0.3.0+2`. Closing.
